proposals/type-classes.md

```diff
@@ -245,7 +253,7 @@ The above instances are each defined alongside their respective type definitions
 
 To determine whether a typeclass definition is a valid type-side implementation we perform the following check:
 
-1. A "local type" is any type (but not typealias) defined in the current file (e.g. everything defined in `data.bar` if we're evaluating `data.bar`).
+1. A "local type" is any type (but not typealias) defined in the current file (e.g. everything defined in `data.bar` if we're evaluating `data.collections.bar`).
```

Comment:
is this accurate?

Reply:
Actually the other reference to `data.bar` should be `data.collections.bar` as well